storage-proofs-core/src/api_version.rs

```diff
@@ -5,6 +5,8 @@ use std::str::FromStr;
 use anyhow::{format_err, Error, Result};
 use semver::Version;
 
+pub use bellperson::groth16::aggregate::AggregateVersion;
```

Comment:
Is this export being used anywhere in the PR?

Reply:
It's explained in the commit message https://github.com/filecoin-project/rust-fil-proofs/pull/1745/commits/c92510e2029188f0ec88cc50e96db0b7410ac4b8:

> `bellperson`s `AggregateVersion` is part of the public API due to the `CompoundProof` trait. With re-exporting it, it's possible that downstream dependants like filecoin proofs API don't need to depend on `bellperson` directly, but can use this re-export instead.